Taxi/Taxi/Presenter/Component/SignUpButton.swift

```diff
@@ -10,33 +10,42 @@ import SwiftUI
 struct SignUpButton: View {
     private let text: String
     private let disabled: Bool
+    private let isLoading: Bool
     private let focusState: Bool
     private let action: () -> Void
 
-    init(_ title: String, _ disabled: Bool = false, focusState: Bool, action: @escaping () -> Void) {
+    init(_ title: String, _ disabled: Bool = false, _ isLoading: Bool = false, focusState: Bool, action: @escaping () -> Void) {
```

Comment:
매개변수명이 너무 추상화되어있다고 생각합니다.
매개변수로 받는 Bool값 중 두 개가 매개변수명 없이 받아지고 있어 각각에 어떤 값을 줘야하는지 알아보기 힘들것 같습니다.
저는 코드가 조금 길어지더라도 명시적인게 낫다고 생각해서 이니셜라이저를 다음과 같이 바꾸는게 더 좋을것 같은데 어떻게 생각하시나요?

```suggestion
    init(_ title: String, isDisabled: Bool = true, isLoading: Bool = false, focusState isFocused: Bool, action: @escaping () -> Void) {
```
```swift
// 현재 사용 예시
SignUpButton(
    "회원가입",
    !nicknameValidation.isValid,
    viewModel.isLoading,
    focusState: nicknameFocusState
)

// 이니셜라이저 변경시
SignUpButton(
    "회원가입",
    isDisabled: !nicknameValidation.isValid,
    isLoading: viewModel.isLoading,
    focusState: nicknameFocusState
)
```

Reply:
> ![](https://user-images.githubusercontent.com/75792767/184088459-16ddca29-80ae-4a34-abce-dea6cbb0ab14.gif) [ ![184088459-16ddca29-80ae-4a34-abce-dea6cbb0ab14.gif](https://user-images.githubusercontent.com/75792767/184088459-16ddca29-80ae-4a34-abce-dea6cbb0ab14.gif) ](https://user-images.githubusercontent.com/75792767/184088459-16ddca29-80ae-4a34-abce-dea6cbb0ab14.gif) [ ](https://user-images.githubusercontent.com/75792767/184088459-16ddca29-80ae-4a34-abce-dea6cbb0ab14.gif)
> 
> 아마도 제가 채팅창에서 뒤로가기 제스처가 가능하게 하려고 만들어놓은 UINavigationControllerExtension 때문에 회원가입 완료화면에서 뒤로가기 제스처가 되는 사소한... 오류가 있습니다 ㅎㅎ 물론 뒤로가기를 해버렸다고 하더라도 회원가입은 이미 된 상태이기 때문에 처음으로 되돌아가서 로그인을 하면 되지만, 혼란에 빠질수도 있을것 같네요. 혹시 막는 방법을 아신다면 막아주세요!🙇‍♂️ 아니면 저 회원가입이 이미 되었다는 알림창 문구를 수정하기만 해도 어느정도 해결될것 같네요. 코드 잘 봤습니다!

introspect Library 를 활용하고, viewDidLoad 이벤트에서 바로 델리게이트 설정을 하는 부분을 제거해 1d67fdbee020855b81e8c52baf5f680ec93d4fba 에서 해당 버그를 수정하였습니다.